src/lib-components/cc-global-header.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+    <header class="vocab header">
+        <nav class="navbar is-default is-active">
+            <div class="navbar-brand">
+                
+            </div>
+            <div class="navbar-menu is-active">
+                <div class="navbar-start">
+                    <!-- Locale options -->
+                    <div class="locale">
+                        <div class="select">
+                            <select>
+                                <option disabled="">Select</option>
+                                <option selected="">English</option>
+                                <option>le Français</option>
+                                <option>日本語</option>
+                                <option>Русский</option>
+                                <option>Español</option>
+                            </select>
+                        </div>
```

Comment:
I'm not sure we want to include/imply a locale selection menu here since the localization issue is still unresolved.

Reply:
Okay, I will just comment it out: in case there is ever a need for it.